reference-implementation/lib/writable-stream.js

```diff
@@ -405,7 +383,7 @@ class WritableStreamDefaultWriter {
     const state = stream._state;
 
     if (state === 'writable') {
-      if (stream._pendingAbortRequest !== undefined) {
+      if (stream._pendingError === true) {
         const error = new TypeError('Requested to abort');
```

Comment:
Error message needs tweaking. Or should we use _storedError? I think that would be consistent with e.g. write().